docs/source/index.rst

```diff
@@ -1,19 +1,54 @@
 Welcome to MyoSuite's documentation!
 =====================================
 
-`MyoSuite <https://sites.google.com/view/myosuite>`_  is a collection of musculoskeletal environments and tasks simulated with the `MuJoCo <http://www.mujoco.org/>`_ physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems
+`MyoSuite <https://sites.google.com/view/myosuite>`_  is a collection of musculoskeletal environments and tasks simulated with the `MuJoCo <http://www.mujoco.org/>`_ physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems.
+
+Check our `github repository <https://github.com/MyoHub/myosuite>`__ for more technical details.
+
+Our paper can be found at: `https://arxiv.org/abs/2205.13600 <https://arxiv.org/abs/2205.13600>`__
+
+Before you start, we highly recommended to have a basic understanding of `OpenAI Gym API <https://gymnasium.farama.org/>`__. Principle knowledge in Reinforcement Learning is also expected for a best experience.
```

Comment:
@raku-slyu @Vittorio-Caggiano - check language here

Reply:
Rephrased to: "Users are expected to have a foundational understanding of Reinforcement Learning and to review the `OpenAI Gym API to have the best experience with Myosuite."

Reply:
Updated based on suggestion

Reply:
Update on commit :  https://github.com/MyoHub/myosuite/pull/192/commits/bdb4236a3dd3560280f1247c874abdab876d76a9